AsyncDisplayKit/ASMapNode.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <MapKit/MapKit.h>
+
+@interface ASMapNode : ASImageNode
+
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
```

Comment:
Hey @appleguy yeah I've been wanting to make this change since the zoom level is currently at 1km. This will make it unambiguous and available to control for everyone. It's going to be a `MKCoordinateRegion`. In terms of the helper functions, what use cases do you envision for this? I'm just asking since you only need a `CLLocationCoordinate2D` and a radius to create a `MKCoordinateRegion` so not sure how helpful it would end up being or what advantages it would have over the built in method?